pyodide_build/recipe/builder.py

```diff
@@ -337,6 +338,33 @@ def _download_and_extract(self) -> None:
         shutil.move(self.build_dir / extract_dir_name, self.src_extract_dir)
         self.src_dist_dir.mkdir(parents=True, exist_ok=True)
 
+    def _override_constraints(self) -> str:
```

Comment:
Maybe let's just call it `_get_constraints()`?

Reply:
Thanks for the suggestion. I renamed it to `_create_constraints_file` as I thought that using the word `get` is confusing regarding that it creates a file internally.